Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -214,28 +253,18 @@ - (void)runDataMigrationIfNeededWithCompletion:(dispatch_block_t)completion {
     }];
 }
 
+- (BOOL)tabBarController:(UITabBarController*)tabBarController shouldSelectViewController:(UIViewController*)viewController {
+    // !!!: disable pop to root behavior
+    return tabBarController.selectedViewController != viewController;
+}
+
 - (void)tabBarController:(UITabBarController*)tabBarController didSelectViewController:(UIViewController*)viewController {
     [self wmf_hideKeyboard];
-
-    WMFAppTabType tab = [[tabBarController viewControllers] indexOfObject:viewController];
```

Comment:
where they were before, when we [configure each tab's root view controller](https://github.com/wikimedia/wikipedia-ios/pull/49/files#diff-bf61ea9e1bae88ed23fb56e10275bea9L55) (you might need to expand the diff.